test/logic/validator_score_v1_logic_test.rb

```diff
@@ -96,6 +96,25 @@ def setup
   end
 
   test 'block_history_get' do
+    Timecop.scale do
```

Comment:
`Timecop.scale` makes milliseconds seem like minutes to ensure these get different `created_at` timestamps

Reply:
https://github.com/travisjeffery/timecop#timecopscale